src/main/java/com/sleekbyte/tailor/Tailor.java

```diff
@@ -47,7 +52,11 @@
  */
 public class Tailor {
 
+    private static AtomicInteger numSkippedFiles = new AtomicInteger(0);
+    private static long numErrors = 0;
```

Comment:
Should all of these counters be of the same type? That is, all `AtomicInteger` or `AtomicLong`?